cycledash/static/js/examine/components/CommentBox.js

```diff
@@ -378,26 +380,28 @@ var VCFCommentEditor = React.createClass({
       );
     }
     buttons.push(
-      <button className='btn btn-xs comment-button btn-success comment-save'
+      <button className='btn comment-button btn-success comment-save'
               key='save'
               onClick={this.handleSaveText}>
         Save
       </button>
     );
     return (
-      <div className='comment-edit-container'>
-        <input className='form-control comment-author'
-               type='text'
-               value={this.state.newAuthorName}
-               placeholder='Enter your name here'
-               onChange={this.handleAuthorChange} />
-        <textarea className='form-control comment-textarea'
-                  value={this.state.newCommentText}
-                  placeholder={this.props.placeHolder}
-                  onChange={this.handleTextChange}
-                  ref='textArea' />
-        <div className='save-buttons'>
-          {buttons}
+      <div className='row'>
```

Comment:
Same explanation as above. It's for the Bootstrap grid. But in time this will probably be redesigned, which would change the markup anyway.